src/net_processing.cpp

```diff
@@ -650,6 +650,14 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
                      "setting. peer=%s version=%s\n",
                 pfrom->GetLogName(), pfrom->cleanSubVer);
         }
+        for (auto entry : pfrom->xVersion.xmap)
+        {
+            auto iter = XVer::keytype.find(entry.first);
+            if (iter != XVer::keytype.end() && iter->second == "c")
```

Comment:
So yeah, with the hypothetical `enum` above, this and the other places would become `iter->second == Xver::keytype_changeable` (use a better naming..) or so.

Reply:
i made a dict to translate the i, c, whatever other letter we use in the .dat to a descriptive word for use in the code.